UltraStar Play/Assets/Common/Playlist/UltraStarPlaylist.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEngine;
+
+public class UltraStarPlaylist
+{
+    private readonly List<UltraStartPlaylistLineEntry> lineEntries = new List<UltraStartPlaylistLineEntry>();
+    public IReadOnlyList<UltraStartPlaylistLineEntry> LineEntries => lineEntries;
+
+    private readonly List<UltraStartPlaylistSongEntry> songEntries = new List<UltraStartPlaylistSongEntry>();
+    public IReadOnlyList<UltraStartPlaylistSongEntry> SongEntries => songEntries;
+
+    public UltraStarPlaylist()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/manual/UltraStar-Deluxe/Play/pullRequest?prid=5574890)